Add missing meta descriptions (#235)
mongodb/docs-php-library: source/reference/method/MongoDBCollection-bulkWrite.txt

MongoDB\\Collection::bulkWrite()
================================

.. meta::

>    :description: Execute multiple write operations in MongoDB using `bulkWrite()` with options for write concern, session, and more.

Reviewer: ```suggestion
   :description: Execute multiple write operations with the MongoDB PHP Library with options for write concern, session, and more.
```
